packages/components/src/font-size-picker/index.js

```diff
@@ -118,6 +126,13 @@ function FontSizePicker(
 		return null;
 	}
 
+	const MaybeVstack = ( { children } ) =>
```

Comment:
This is needed to maintain the spacing before the slider in the `withSlider` case 😅

Reply:
Thank you! It would have taken me a while to understand, without this comment :)